server/src/main/java/org/candlepin/policy/js/compliance/StatusReasonMessageGenerator.java

```diff
@@ -64,8 +64,9 @@
         REASON_MESSAGES.put(ComplianceReason.ReasonKeys.VIRT_CPU,
             I18n.marktr("Only supports {1} of {2} vCPUs."));
         REASON_MESSAGES.put(ComplianceReason.ReasonKeys.UNMAPPED_GUEST,
-            I18n.marktr("Guest has not been reported on any host and is using a temporary unmapped guest " +
-            "subscription."));
+            I18n.marktr(
+            "Guest has not been reported on any host and is using a temporary unmapped guest subscription." +
+            " For more information, please read here: https://access.redhat.com/solutions/1592573"));
```

Comment:
To sound a little more natural, I would phrase this last sentence:

"For more information, please see https://access.redhat.com/solutions/1592573"